test/infra/control/run-tests-isolated.bash

```
@@ -431,6 +441,9 @@ docker run \
             exit 1
         fi
 
+        echo '>>> Waiting for binlog readers to be ready...'
+        sleep "\${BINLOG_READER_START_DELAY}"
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)